src/components/todo/list.js

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
+import { ListGroup } from 'react-bootstrap';
+
+
+const TodoList = (props) => {
+  return (
+    <>
+
+
+<ListGroup>
+  <ListGroup.Item>Cras justo odio</ListGroup.Item>
+  <ListGroup.Item>Dapibus ac facilisis in</ListGroup.Item>
+  <ListGroup.Item>Morbi leo risus</ListGroup.Item>
+  <ListGroup.Item>Porta ac consectetur ac</ListGroup.Item>
+  <ListGroup.Item>Vestibulum at eros</ListGroup.Item>
```

Comment:
I would like to see all of these deleted out of here, it's a placeholder for you. What you should be doing in this `ListGroup` is the functionality that you have in your render below. When you do that as well you don't have to create a `<ul>` or `<li>` you can use that `<ListGroup.Item>`

Reply:
Will do! 
I struggled with losing functionality when implementing ListGroup and ListGroup.Item, so I kind of left it there as a stepping stone to remind me of what to do next.